src/components/ui/Input.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+
+const Input = (props) => {
+    const {
+        handleChange,
+        value,
+        id,
+        name,
+        type,
+        isRequired = false,
+        placeholder,
+        label,
+        position = "top",
+        labelFontSize = "5xl",
+        labelFontWeight = "normal",
+        radius = "md",
+        inputWidthSize = "",
+        inputHightSize = "",
+        inputFontSize = "3xl",
+        inputFontWeight = "light",
+
+        ...rest
+    } = props;
+
+    const inputContainerClasses =
+        position === "left" || position === "right"
+            ? "flex items-center "
+            : "block ";
```

Comment:
We can control direction of the label by styling the parent-div surrounds Input component> Parent container should be flexed or block.
So, you can remove these lines